analyzer/tests/functional/analyze_and_parse/test_analyze_and_parse.py

```diff
@@ -65,7 +65,9 @@ def setup_class(cls):
         cls.test_workspaces = {'NORMAL': os.path.join(test_workspace,
                                                       'NORMAL'),
                                'CHECK': os.path.join(test_workspace,
-                                                     'CHECK')}
+                                                     'CHECK'),
+                               'OUTPUT': os.path.join(test_workspace,
+                                                      'OUTPUT')}
```

Comment:
In our function tests the generated files stored in the "test workspace" temporary directory what the test module init clean.  This commit follows that pattern. In this test suite the temporary directories are managed by the cls.test_workspaces dictionary. I follow this way instead of introducing a new class specific variable that points to the suite specific directory wehere the temporary output files generated to.

Reply:
Your suggestion was implemented.